homeworks/serhii.noskov_SerhiiNSN/5-logic/homework.js

```diff
@@ -0,0 +1,47 @@
+/**
+ *
+ * @param firstValue string with first number
+ * @param secondValue string with second number
+ * @param operation string with operation symbol
+ * @returns {string|number} result of calculation or error message:
+ * - if firstValue or secondValue is not a number, return 'Enter a number'
```

Comment:
Hey, @SerhiiNSN, here should be an error 
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/65feade5-1cfe-420c-95cc-601bcbcc9ad6)